lib/audit/checksum.rb

```diff
@@ -54,16 +54,18 @@ def self.validate_druid(druid)
     start_msg = "#{Time.now.utc.iso8601} CV validate_druid starting for #{druid}"
     puts start_msg
     Rails.logger.info start_msg
-    po = PreservedObject.find_by(druid: druid)
-    pres_copy = PreservedCopy.find_by(preserved_object: po)
-    endpoint_name = pres_copy.endpoint.endpoint_name
-    cv = ChecksumValidator.new(pres_copy, endpoint_name)
-    cv.validate_checksums
+    # po = PreservedObject.find_by(druid: druid)
+    # pres_copy = PreservedCopy.find_by(preserved_object: po)
```

Comment:
👍 overall, but looks like a bit of debris from the old version is still there 🙂

Reply:
you always catch me